src/Xamarin.Android.Build.Tasks/Xamarin.Android.Legacy.targets

```diff
@@ -12,6 +12,7 @@ projects. .NET 5 projects will not import this file.
 
   <PropertyGroup>
     <AndroidBoundExceptionType Condition=" '$(AndroidBoundExceptionType)' == '' ">Java</AndroidBoundExceptionType>
+    <AndroidGenerateResourceDesigner>True</AndroidGenerateResourceDesigner>
```

Comment:
I think this might need to check if it is blank, like it does in .NET 6:

https://github.com/xamarin/xamarin-android/blob/d4da1c252f45c4910abc1bd9e5be9ecf9dc683a0/src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets#L13

This way you can set it to `false`.

Reply:
We don't support this property under Legacy as per the [docs](https://github.com/xamarin/xamarin-android/blob/b21cbf943a19079773f9e762908939181a7f2180/Documentation/guides/building-apps/build-properties.md#androidgenerateresourcedesigner). So I think it should always default to `true`

Reply:
If you set to `false` in your `.csproj`, this would change it to `true`. I think we want to allow them to set it to `false`?